crates/turbopack-ecmascript-runtime/js/src/dev/runtime/none/runtime-backend-none.ts

```diff
@@ -22,16 +22,8 @@ type ExternalRequire = (
 ) => Exports | EsmNamespaceObject;
 type ExternalImport = (id: ModuleId) => Promise<Exports | EsmNamespaceObject>;
 
-interface TurbopackDevContext extends TurbopackDevBaseContext {
-  x: ExternalRequire;
-  y: ExternalImport;
-}
-
 function augmentContext(context: TurbopackDevBaseContext): TurbopackDevContext {
-  const nodejsContext = context as TurbopackDevContext;
-  nodejsContext.x = externalRequire;
-  nodejsContext.y = externalImport;
-  return nodejsContext;
```

Comment:
This change breaks the edge runtime, which actually can have externals, e. g. `require("node:buffer")`

Reply:
We should probably rename runtime-none to runtime-edge...